### PR TITLE
[HOLD] Update assets script

### PR DESCRIPTION
**Pull request by eessex**
Uses the `yarn webpack` command in assets rather than pointing to node modules, which ensures we pass all expected options (including config) to webpack.

### CODE REVIEW

File: scripts/assets.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -e -x
 
 yarn clean
 
-NODE_ENV=production node_modules/.bin/webpack
+NODE_ENV=production yarn webpack
 stylus \
```
Comment (eessex) [inline scripts/assets.sh]: @damassi wanted to check -- now that we are using the assets compiled by webpack, is the stylus compression here necessary?

File: webpack/environments/prodConfig.js [hunk 2]
```diff
@@ -24,8 +24,6 @@ module.exports.prodConfig = {
   optimization: {
     minimizer: [
       new TerserPlugin({
-        cache: false,
```
Comment (eessex) [inline webpack/environments/prodConfig.js]: not sure why, but this change did not seem to make it into the last pr

File: scripts/assets.sh (hunk 1 above)
Comment (damassi) [reply scripts/assets.sh]: The way it works in force is that when we started using the asset manifest plugin (which was a PR _after_ the webpack refactor) we build and [seed the CSS here](https://github.com/artsy/force/blob/master/webpack/envs/productionConfig.js#L29). 

I think removing this is for a later PR, once bucket-assets is ready to be removed completely along with its asset hashing.

File: src/lib/assetMiddleware.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk"
+import fs from "fs"
+import { isProduction } from "./environment"
+import path from "path"
+
+const { CDN_URL } = process.env
+
+export const assetMiddleware = () => {
```
Comment (damassi) [inline src/lib/assetMiddleware.js]: Before adding the asset middleware we'll want to make sure the current webpack stuff in master is working 100%. In force I did things in two phases; 1) make webpack setup a little friendlier to changes over time; 2) remove bucket assets. 

These were the stages (with maybe a few intermediate prs inbetween that fixed issues): 
1) https://github.com/artsy/force/pull/3434
2) https://github.com/artsy/force/pull/3458
3) https://github.com/artsy/force/pull/3630

I would reference the Arena PR linked in the ticket, however. It's straight forward and to the point.

**Pull request closed without merging**